src/main/java/gmail/mishchuk/lesson38_Map/MapUtils.java

```diff
@@ -0,0 +1,25 @@
+package gmail.mishchuk.lesson38_Map;
+
+import java.util.*;
+
+public class MapUtils {
+
+    public static Map<String, Integer> getUnitedMap(Map<String, Integer> map1, Map<String, Integer> map2) {
+        Map<String, Integer> finalMap = new LinkedHashMap<>(map1);
```

Comment:
Можна просто `HashMap`. Порядок нас не сильно цікавить в даному випадку
```suggestion
        Map<String, Integer> finalMap = new HashMap<>(map1);
```